internal/gatewayapi/conformance/support_level.go

```diff
@@ -26,10 +26,18 @@ const (
 )
 
 // ExtendedFeatures is a list of supported Gateway-API features that are considered Extended.
-var ExtendedFeatures = sets.New[features.SupportedFeature]().
```

Comment:
why do we need to change this ?

Reply:
Previously `features.Feature` was a string, but it's been changed to a structure in Gateway API v1.2.0